lib/util/version.js

```diff
@@ -14,8 +14,19 @@ function getFromContext(context) {
   return confVer.split('.').map(part => Number(part));
 }
 
-function test(context, methodVer) {
-  const confVer = getFromContext(context);
+function getFlowVersionFromContext(context) {
+  let confVer = '999.999.999';
+  // .eslintrc shared settings (http://eslint.org/docs/user-guide/configuring#adding-shared-settings)
+  if (context.settings.react && context.settings.react.flowVersion) {
+    confVer = context.settings.react.flowVersion;
+  } else {
+    throw 'Could not retrieve flowVersion from settings';
```

Comment:
Yes. That's what we wanted, right? 

We don't want to provide a default value, so `test()` cannot return `true` or `false` in case it is not specified. Therefore, throwing an exception instead which we then handle in the calling code by catching it and using the default logic.